Recipes/Recipes/CategoryModule/View/CategoryViewCell.swift

```diff
@@ -147,48 +247,38 @@ final class CategoryViewCell: UITableViewCell {
         ])
     }
 
-    private func setupTimerRecipeImageConstraints() {
+    private func setupPointerRecipeImageConstraints() {
         NSLayoutConstraint.activate([
-            timerRecipeImage.leadingAnchor.constraint(equalTo: photoRecipeImage.trailingAnchor, constant: 20),
-            timerRecipeImage.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -23),
-            timerRecipeImage.heightAnchor.constraint(equalToConstant: 15),
-            timerRecipeImage.widthAnchor.constraint(equalToConstant: 15)
+            pointerRecipeImage.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -15.42),
+            pointerRecipeImage.centerYAnchor.constraint(equalTo: photoRecipeImage.centerYAnchor),
+            pointerRecipeImage.heightAnchor.constraint(equalToConstant: 20),
+            pointerRecipeImage.widthAnchor.constraint(equalToConstant: 12.35)
         ])
     }
 
-    private func setupTimeRecipeLabelConstraints() {
-        NSLayoutConstraint.activate([
-            timeRecipeLabel.leadingAnchor.constraint(equalTo: timerRecipeImage.trailingAnchor, constant: 4),
-            timeRecipeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -23),
-            timeRecipeLabel.heightAnchor.constraint(equalToConstant: 15),
-            timeRecipeLabel.widthAnchor.constraint(equalToConstant: 55)
-        ])
-    }
+    private func createAnimationGroup(previousGroup: CABasicAnimation? = nil) -> CABasicAnimation {
+        let animDuration: CFTimeInterval = 1.5
 
-    private func setupKalRecipeImageConstraints() {
-        NSLayoutConstraint.activate([
-            kalRecipeImage.trailingAnchor.constraint(equalTo: kalRecipeLabel.leadingAnchor, constant: -5),
-            kalRecipeImage.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -23),
-            kalRecipeImage.heightAnchor.constraint(equalToConstant: 15),
-            kalRecipeImage.widthAnchor.constraint(equalToConstant: 15)
-        ])
-    }
+        let basicAnimation = CABasicAnimation(keyPath: #keyPath(CAGradientLayer.locations))
+        basicAnimation.fromValue = [-1.0, -0.5, 0]
+        basicAnimation.toValue = [1, 1.5, 2]
+        basicAnimation.duration = animDuration
+        basicAnimation.beginTime = 0.0
+        basicAnimation.repeatCount = .infinity
 
-    private func setupKalRecipeLabelConstraints() {
-        NSLayoutConstraint.activate([
-            kalRecipeLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -60),
-            kalRecipeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -23),
-            kalRecipeLabel.heightAnchor.constraint(equalToConstant: 15),
-            kalRecipeLabel.widthAnchor.constraint(equalToConstant: 72)
-        ])
+        if let previousGroup = previousGroup {
+            basicAnimation.beginTime = previousGroup.beginTime + 0.33
```

Comment:
```suggestion
            basicAnimation.beginTime = previousGroup.beginTime +
```
это уже магические числа